src/app/cluster/details/cluster/mla/component.ts

```diff
@@ -31,6 +31,6 @@ export class MLAComponent {
   @Input() addons: Addon[];
 
   isLoadingData(): boolean {
-    return _.isEmpty(this.alertmanagerConfig) || _.isEmpty(this.ruleGroups) || !this.isClusterRunning;
+    return !this.isClusterRunning;
```

Comment:
Please use it directly in the template instead.

Reply:
👍